src/utils/Strings.ts

```diff
@@ -316,6 +317,14 @@ class Strings {
 
     static disabledSegment = "Deshabilitar horario"
     static enableSegment = "Habilitar horario"
+
+    //Reports
+    static started_at = "Fecha de Inicio"
+    static finished_at = "Fecha de Fin"
+
+    static appVersion = 'v.1.0.41.10-PROD';
```

Comment:
di think that we don;t use this appVersion please remove it

Reply:
I have already eliminated